### PR TITLE
Add type definition for IPC Event on main process

### PR DESCRIPTION
**Pull request by fabon-f**
http://electron.atom.io/docs/v0.36.0/api/ipc-main/#ipc-event

### CODE REVIEW

File: github-electron/github-electron.d.ts [hunk 1]
```diff
@@ -1464,6 +1464,24 @@ declare module GitHubElectron {
...
+		setMaxListeners(n: number): IPCMain;
+		getMaxListeners(): number;
+		listeners(event: string): Function[];
+		emit(event: string, ...args: any[]): boolean;
+		listenerCount(type: string): number;
+		on(event: string, listener: (event: IPCMainEvent, ...args: any[]) => any): IPCMain;
+	}
+
+	interface IPCMainEvent {
+		returnValue?: any;
```
Comment (rhysd) [inline github-electron/github-electron.d.ts]: Why not `returnValue: any;`?  When `returnValue` member is omitted? I have the same question for below `sender` member.

Comment (hinamiyagk) [inline github-electron/github-electron.d.ts]: https://github.com/atom/electron/blob/v0.36.2/atom/browser/api/lib/web-contents.coffee#L65-L71
`event.returnValue` is not a normal member(it has only setter).
Assignment to `event.returnValue` has no effect on asynchronous events. 
But `sender` Member is always available. I'll fix it.

Comment (rhysd) [inline github-electron/github-electron.d.ts]: I understood `event.returnValue`. Thanks.

**Comment by rhysd** [comment]
LGTM :+1:

**Comment by jedmao** [comment]
:+1:

**Comment by elisee** [comment]
Please make sure to merge #7164 (which I just rebased for the second time) first!

**Comment by vvakame** [comment]
@elisee :wink: 
@hinamiyagk please resolve conflict.

**Comment by hinamiyagk** [comment]
How can I resolve conflict?

**Comment by elisee** [comment]
@hinamiyagk yes rebasing should do it

**Comment by hinamiyagk** [comment]
I closed this PR by mistake. I'll open same PR again.

**Pull request closed without merging**